logging.conf

```
@@ -0,0 +1,27 @@
+[loggers]
+keys=root, transaction
+
+[handlers]
+keys=consoleHandler
+
+[formatters]
+keys=simpleFormatter
+
+[logger_root]
+level=INFO
+handlers=consoleHandler
+
+[logger_transaction]
+level=INFO
+handlers=consoleHandler
+qualname=transaction
+propagate=0
+
+[handler_consoleHandler]
+class=StreamHandler
+level=INFO
+formatter=simpleFormatter
+args=()
+
+[formatter_simpleFormatter]
+format=%(asctime)s - %(name)s - %(levelname)s - %(message)s
```

Please, get rid of this conf file and replace it by happy_bank_core/config/log.py module that contains the following code:
```suggestion
import logging


def setup_logging():
    logging.basicConfig(
        level=logging.DEBUG,
        format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
    )

```

That should be enough and unit tests still pass.